front/src/applications/stdcm/components/SimulationReportSheetV2.tsx

```diff
@@ -10,21 +10,19 @@ import { formatDateToString, formatDayV2 } from 'utils/date';
 
 import styles from './SimulationReportStyleSheet';
 import type { SimulationReportSheetProps } from '../types';
-import { extractSpeedLimit, getStopDurationTime, getOperationalPointsWithTimes } from '../utils';
+import { extractSpeedLimit, getStopDurationTime } from '../utils';
 
 const SimulationReportSheetV2 = ({
   stdcmData,
-  pathProperties,
-  rollingStockData,
-  speedLimitByTag,
   simulationReportSheetNumber,
   mapCanvas,
-  creationDate,
+  operationalPointsList,
 }: SimulationReportSheetProps) => {
   const { t } = useTranslation('stdcm-simulation-report-sheet');
   let renderedIndex = 0;
 
```

Comment:
To improve again a bit the code, you can extract rollingStock, speedLimitByTag, creationDate and departure_date from stdcmData:
```suggestion
 
   const { rollingStock, speedLimitByTag, departure_time: departureTime, creationDate } = stdcmData;
```

Reply:
fix